src/main/java/core/basesyntax/vehicles/Truck.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.vehicles;
+
+import core.basesyntax.Machine;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");
```

Comment:
Similarly, the message 'Truck stopped working' should be more informative. Consider specifying what the Truck has finished doing, such as 'Truck has completed transporting goods'.